pkg/maps/lbmap/lbmap.go

```diff
@@ -223,7 +223,7 @@ func deleteServiceProto(svc loadbalancer.L3n4AddrID, backendCount int, useMaglev
 		}
 	}
 
-	if useMaglev {
+	if useMaglev && backendCount > 0 {
 		if err := deleteMaglevTable(ipv6, uint16(svc.ID)); err != nil {
```

Comment:
Lgtm. Is there any harm in always calling deleteMaglevTable() but w/o the err check?

Reply:
I'd keep the error check. Otherwise, we are risking to end up in a situation in which we don't have a good understanding what's expected and what's not.